schemas/error.rng

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
wonder where we can put a link to the original file, since this is probably lifted from `schemas` but it's not obvious

Reply:
I can try to pull the `libero/schemas` repo and see if I can read it directly. What do you think?

Reply:
I'm going to create an issue to do this separately after this PR is merged.